### DISCUSSION TITLE
Could not bind to 0.0.0.0:0

### DISCUSSION THREAD

**Discussion by schmale96** [dannagle/PacketSender#323] [Q&A]
Hello,

I have a question. Does anyone know this error: Could not bind to 0.0.0.0:0

I get this message when I call the MS-DOS application packetsender.com with the CMD..

Thanks in advance.

**Comment by dannagle** [comment]
Have you tried it with `-4` or `--ipv4` or `--bindip YOUR_IP` ?

**Reply by schmale96** [reply]
This is the CMD order I tried to execute: packetsender --bindip 10.149.137.183 --tcp 172.30.16.55 23 "FF FA 2C 32 12 FF F0" -w 4000

The first IP I want to bind is the IP of the sending server is this correct? Now it says "Could not bind to 10.149.137.183"

**Reply by dannagle** [reply]
You may not have that IP address to bind to. You should not need to bind for TCP unless you have 2 overlapping addresses and an obscure need to specify which one to use. 

Try the command just specifying the data and port and let the defaults do the work.

Here is reworked example using your telnet data. I'm targeting an online public server. Adapt this command as needed. Note I am using powershell which requires `./`. 
```
PS C:\Program Files\PacketSender> ./packetsender.com avalon-rpg.com 23 "FF FA 2C 32 12 FF F0" -w 4000
```

**Reply by schmale96** [reply]
This worked now, thank you for yor help.
